src/ts/forwarders/forwarderFactory.ts

```diff
@@ -20,11 +20,11 @@ export abstract class ForwarderFactory<D> {
    * @param replayProtectionType Replay Protection
    * @param signer Signer's wallet
    */
-  public abstract async createNew(
+  public abstract createNew(
     chainid: ChainID,
     replayProtectionType: ReplayProtectionType,
     signer: Wallet
-  ): Promise<D>;
+  ): D;
```

Comment:
why is there a sad face

Reply:
https://www.youtube.com/watch?v=d-diB65scQU

Reply:
lol